README.md

```diff
@@ -65,6 +65,33 @@ _Default_: `"widely"`
 - `"newly"` – Allows features supported in all Baseline browsers for less than 30 months. Limited availability features still trigger warnings.
 - `YYYY` – Allows features that became Baseline newly available that year, or earlier. For example, `2023`.
 
+### `ignoreSelectors`: `Array<string | RegExp>`
+
+_Default_: `[]`
+
+An array of selectors to ignore. Each item can be:
+
+- A string for exact matches
+- A regular expression (e.g., `/^has/`)
```

Comment:
Could you format it like the following?

```
Given:

- An example configuration of the rule.

The following patterns are not considered problems:

- Patterns that are ignored by the rule (around two types).
```
ref: https://github.com/stylelint/stylelint/tree/22ca9e673f07ea57167f0f655ab1559bb08feec7/lib/rules/at-rule-no-deprecated#ignoreatrules

Reply:
Please handle the other options in the same way.